### PR TITLE
Make aliasexecution endpoint return 201

### PR DESCRIPTION
**Pull request by enykeev**
NO MERGE

### CODE REVIEW

File: st2api/st2api/controllers/v1/aliasexecution.py [hunk 1]
```diff
@@ -45,7 +46,7 @@
 
 class ActionAliasExecutionController(rest.RestController):
 
-    @jsexpose(body_cls=AliasExecutionAPI, status_code=http_client.OK)
+    @jsexpose(body_cls=AliasExecutionAPI, status_code=http_client.CREATED)
```
Comment (lakshmi-kannan) [inline st2api/st2api/controllers/v1/aliasexecution.py]: CREATED seems like the right code.

Comment (Kami) [inline st2api/st2api/controllers/v1/aliasexecution.py]: Yeah POST /executions endpoint also returns 201.

File: st2api/st2api/controllers/v1/aliasexecution.py [hunk 2]
```diff
@@ -87,7 +88,7 @@ def post(self, payload):
                                              notify=notify,
                                              context=context)
 
-        return str(execution.id)
+        return execution
```
Comment (Kami) [inline st2api/st2api/controllers/v1/aliasexecution.py]: Yeah, that's not consistent with /executions endpoint as well.

Thanks.

**Comment by Kami** [comment]
Will we also need to update / make related changes to the hubot plugin?

**Comment by Kami** [comment]
Thanks.

So far the changes look good to me (since this is backward incompatible change we will also need to update upgrade notes, maybe also hubot-stackstorm plugin and add tests, but I assume you are already aware of this :))

File: st2api/st2api/controllers/v1/aliasexecution.py (hunk 2 above)
Comment (manasdk) [inline st2api/st2api/controllers/v1/aliasexecution.py]: I am happy to see this changed but what is the usecase? e.g. why do we need more than an execution-id at this point?

The need for consistency with /executions is not obvious to me - can you guys explain why it is desired?

File: st2api/st2api/controllers/v1/aliasexecution.py (hunk 1 above)
Comment (manasdk) [inline st2api/st2api/controllers/v1/aliasexecution.py]: +1

**Comment by enykeev** [comment]
@Kami yeah, actually, https://github.com/StackStorm/hubot-stackstorm/pull/49 already supports both ways.

**Comment by emedvedev** [comment]
Included in #2229 (along with updated tests for 201), closing.

**Pull request closed without merging**